### DISCUSSION TITLE
Auto Feed Pets - First Order

### DISCUSSION THREAD

**Discussion by Eric-Burdo** [bumbleshoot/automate-habitica#33] [Suggestions]
An enhancement for the Auto Feed/Hatch script.

I'm at the point where I have all the pets, and I'm just cycling the "Keys to the Kennel" and then hatching the pets.

I'd like an enhancement that I can set a setting, and it just picks the first available "favorite food" for a pet and feeds it... and cycles that until I have no food left.

Similar with hatching.  If there is an unhatched pet and I have eggs... do it.

**Comment by purchace** [comment]
I think this is the same issue I'm having -- I found this, purely to feed any non feed pets into mounts. I don't think it seems to do that, if it's not a basic pet?
